src/pin-field.tsx

```diff
@@ -152,7 +152,7 @@ const PinInput = ({
     };
 
     return (
-        <Inline space={8}>
+        <Inline space={8} wrap verticalSpace={16}>
```

Comment:
This is the only change needed in `PinField`, but needed changes in `Inline` to support this